pkgs/applications/editors/emacs/elisp-packages/yes-no/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchurl, trivialBuild }:
+
+trivialBuild {
+  pname = "yes-no";
+  version = "0";
```

```suggestion
  version = "2017-10-01";
```

Given that this is a spare file, the version number into it makes few to no sense. Indeed it was modified two times and the Version line wasn't updated.

---

Updated! Thanks!